src/interfaces/node.cpp

```diff
@@ -218,6 +220,18 @@ class NodeImpl : public Node
         LOCK(::cs_main);
         return ::pcoinsTip->GetCoin(output, coin);
     }
+    std::string getWalletDir() override
+    {
+        return GetWalletDir().string();
+    }
+    std::vector<std::string> listWalletDir() override
+    {
+        std::vector<std::string> paths;
+        for (auto& path : ListWalletDir()) {
+            paths.push_back(path.string());
+        }
+        return paths;
```

Comment:
Sort alphabetically?

Reply:
I think that's something for the UI to handle. It can sort by other attribute, like last modified, transaction count, balance, etc.

Reply:
That's fine, as long as the RPC interface does it, that's also a User Interface :-)

Reply:
Should we sort `listwallets` result too?

Reply:
That could be a nice improvement. Not necessarily in this PR, but perhaps there could be a common function to render the list of wallets in RPC, to make sure they're consistent.

Reply:
Should it ignore case? Should have an option to support numeric/date prefixes?

From the cli point of view, you can do stuff like `bitcoin-cli -regtest listwalletdir | jq -r '.wallets | map(.name) | join("\n")' | sort ...`.

Reply:
Don't most alphabetical sort functions ignore case (unless it's a tie) and put numbers before letters? I'm a fan of `jq`, but seems overkill for a simple list. It doesn't seem terribly complicated in C++11 even with [UTF-8](https://www.quora.com/How-do-I-sort-an-array-of-strings-in-alphabetical-order-in-C) (1st answer, 3rd example):

```
std::wstring a[] = {L"Шла", L"Саша", L"по", L"шоссе", L"и", L"сосала", L"сушку"};
std::sort(std::begin(a), std::end(a), std::locale("en_US.utf8"));
std::wstring_convert<std::codecvt_utf8<wchar_t>> cvt;
for(auto& s: a) std::cout << cvt.to_bytes(s) << ' ';
```

Reply:
If you don't mind I prefer to follow up that discussion and possible implementation in both RPC. In this PR I'm just following `listwallets` ordering and for now having the functionality is enough.